### ISSUE TITLE
Web: don't show that easily the message "dispute template not correct, please vote refuse to arbitrate"

### ISSUE THREAD

**Issue by kemuru** [kleros/kleros-v2#1459]
![2024-01-26 18 15 03](https://github.com/kleros/kleros-v2/assets/102478601/38205ae4-f17e-4173-8dd4-e4e2bbc1c7e5)

for this user, it showed "invalid dispute template, please vote refuse to arbitrate" when it was a lie, it was just that it didn't fetch properly probably. his testimony:

"That message is problematic because sometimes the rpc fails and the case does not load properly, but the solution in the first place is not to vote refuse to arbitrate but to wait for them to fix the rpc to see the case. A juror who enters at the beginning of the voting period and encounters that message votes refuse, and the next day, the other jurors enter and vote correctly. Whoever voted first loses their pnk, and I'm not speaking hypothetically, it has happened.For me you don't have to tell the jury how to vote specifically. A message could be: if the template remains broken until near the end of voting period, please consider voting refuse to arbitrate. It has to be clear when the template is broken as well."

later, I checked and it loaded correctly for me:

<img width="1137" alt="Screenshot 2024-01-26 at 16 22 41" src="https://github.com/kleros/kleros-v2/assets/102478601/52a2602e-9e66-40a6-9dcb-9d7525abfac8">

so in conclusion, we are showing the message "please vote refuse to arbitrate" too easily and it is misleading sometimes.

**Comment by Harman-singh-waraich** [comment]
Same as #1453 ?
